insights/specs/datasources/ls.py

```diff
@@ -52,7 +54,13 @@ def list_with_la_filtered(broker):
 
 @datasource(HostContext)
 def list_with_lan(broker):
-    return _list_items(Specs.ls_lan_dirs)
+    filters = _list_items(Specs.ls_lan_dirs)
+    if 'specs.fstab_mounted' in filters and FSTab in broker:
+        filter_list = filters.split()
+        for mntp in broker[FSTab].mounted_on.keys():
+            mnt_point = os.path.dirname(mntp)
+            filter_list.extend([mnt_point] if mnt_point and mnt_point not in filter_list else [])
+    return ' '.join(filter_list)
```

Comment:
There could be a case that there are more than 1k mount entries from mount command. But not known how's the mount entries count in /etc/fstab. If the mount_point got a quite large count than avg, say, 100, will this `ls -lan` command work?

Reply:
Testing result with 5003 mount in /etc/fstab:
```
{"name": "insights.specs.Specs.ls_lan", "exec_time": 0.0006165504455566406, "errors": [], "results": {"type": "insights.core.spec_factory.CommandOutputProvider", "object": {"rc": null, "cmd": "...
```
the ls command work well